vault/core.go

```diff
@@ -1146,8 +1146,6 @@ func (c *Core) sealInitCommon(ctx context.Context, req *logical.Request) (retErr
 		return retErr
 	}
 
-	req.SetTokenEntry(te)
```

Comment:
This was the only instance that I found where `req.SetTokenEntry` was called after a `fetchACLTokenEntryAndEntity` call.